en/django_start_project/README.md

```diff
@@ -198,6 +234,10 @@ If you're using a Chromebook and Cloud9, instead click the URL in the pop-up win
 ```
 https://<a bunch of letters and numbers>.vfs.cloud9.us-west-2.amazonaws.com
 ```
+or on Glitch :
```

Comment:
Putting a space before a colon (`:`) is AFAIK a French convention not usually followed in English.
```suggestion
or on Glitch:
```